examples/api/demoapp/demo.go

```diff
@@ -1,15 +1,15 @@
 package main
 
 import (
-	"bufio"
+    "bufio"
```

Comment:
Thought I had that figured out. Apparently not. Will add a commit momentarily. Thanks.